docs/kb/semgrep-supply-chain/scanning-projects-with-multiple-lock-files.md

```diff
@@ -0,0 +1,108 @@
+---
+tags:
+  - GitHub Actions
+  - Lock files
+description: Scanning projects with multiple lock files in GitHub Actions
+---
+
+# Scanning projects with multiple lock files in GitHub Actions
+
+Your project may have many modules, each with a dependency descriptor file; for example, `pom.xml` if Maven is used as a build dependency tool.
```

Comment:
For "dependency descriptor file" do you mean a manifest file or a lock file?

Reply:
For example, for Maven I would say:
- pom.xml is a dependency descriptor file or manifest file
- maven_dep_tree.txt is the lock file

Reply:
Ok, I'll just suggest to prefer "manifest file" because it's more common :)